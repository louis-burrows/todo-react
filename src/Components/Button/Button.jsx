import React from 'react';
import styles from './Button.module.scss';

const Button = (props) => {
  
  const {text, handleClick} = props;

  return ( 

    <>
      <button 
        className={styles.standardButton} 
        onClick={handleClick}
      >
        {text}
      </button>
    </>

  );
}
 
export default Button;