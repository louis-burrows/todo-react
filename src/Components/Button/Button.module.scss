.standardButton {
  width: 100px;
  height: 25px;
  border-radius: 5px;
  border: solid black 1px;
  &:hover {
    cursor: pointer;
    background-color: rgb(240, 255, 253);
  }
}