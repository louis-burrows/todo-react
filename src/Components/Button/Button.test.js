import React from "react";
import { shallow } from "enzyme";
import Button from "./Button";

describe("Button tests", () => {

  let testComponent;

  beforeEach(() => {
    testComponent = shallow(<Button text={"test text"}/>);
  })

  it("should render", () => {
    expect(testComponent).toBeTruthy();
  });

});
