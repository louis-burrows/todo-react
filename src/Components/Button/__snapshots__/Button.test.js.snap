// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button tests should match the snapshot 1`] = `ShallowWrapper {}`;
